server/redis.py

```diff
@@ -4,18 +4,21 @@
 from uvicorn.config import logger
 
 from .environment import REDIS_HOST, REDIS_LOCAL_PORT
-from server.websockets import (
-    notify_error_to_client,
-    notify_feedback,
-)
 
 from typing import Dict
+from server.constants import REDIS_ERROR
 
 
-r = redis.Redis(host=REDIS_HOST, port=REDIS_LOCAL_PORT, db=0, charset="utf-8", decode_responses=True)
+r = redis.Redis(
```

Comment:
I would change this variable name to something more meaningful

Reply:
Done, changed from r to redis_data.